### ISSUE TITLE
request for easyeda files

### ISSUE THREAD

**Issue by irakhlin** [IainPHay/ESPHome-bed-occupancy-FSR#2]
Hi,

If you used easyeda for this project could you share the project files if you still had them available? I wanted to add/ make some modifications and it would really save me time on the esp32 footprint which I cant seem to find.

Thank you

**Comment by IainPHay** [comment]
Sure did, have uploaded for you, interested to see what you have planned. I've been meaning to do an update for a while as run ESPhome proxy on it and iwatch detection and all sorts now. Planning to add mmwave radar sensor etc

**Comment by gbosky** [comment]
Just came across this project to see what others were doing. I have been having issues with selecting resistors that work with both sides of the mattress I use. Its older firm/foam mattress that seems to curl on the ends and throws of the weight/detection of pressure when on or off. Definitely need to build a PCB with trim pots so I can tune it on the fly.

What kind of mattress do you use? I would think some of the firm mattresses would be heavier than spring mattress and not sure if 10kOhm would work in my case. May go up to 25kOhm.

Also for any others that may read this in the future I may look into building a similar PCB but one that works with a ADS 1115 sensor from adafruit. (https://www.adafruit.com/product/1085) I already have a ESP that drives my bed trim LED strips with an I2C connection to this sensor. Currently have a Proto-board PCB but would like something more clean like what you have done with trim pots and direct connections to the FSR!

**Comment by IainPHay** [comment]
I use an Ikea mattress which is spring but really heavy! I used 100k trim pots and on one side it is set to about 30K and on the other about 13k the trim pots I used are linear and super fine adjustment can be done so don’t mess around just go for 100K. Once it’s set it’s a really clean solution and I’ve not had to adjust it for months. I have found that once the bedding has been changed it’s a good idea to sit above the fsr and rock around for a second or 2 as this seems to get it right back to stable voltage before sleep.

**Comment by irakhlin** [comment]
@IainPHay 

Thank you sorry for not updating earlier. I used the project files as a sanity check of sorts as I typically like to do with easyeda/jlcpcb projects especially when using footprints from their "community" library.

As for the changes, basically I just remade your design but for the 38pin ESP32 nodemcu board,  could not find any source of the 36pin boards you used outside of 1+ month shipping times but had a whole bin of the 38 pin boards. I also had no use for the I2C sensor as each of my rooms already has more than enough temperature sensors so it was not necessary for my use case. Lastly I had a ton of credits and coupons for jlcpcb assembly so I just created the project with the needed pick and place files. Got them in a week or so ago and the first dual sensor one went into my 6 month old son's crib and has been working great!
